mappings/net/minecraft/entity/passive/FoxEntity.mapping

```diff
@@ -0,0 +1,98 @@
+CLASS ana net/minecraft/entity/passive/FoxEntity
+	CLASS ana$a DefendFriendGoal
+		FIELD k offender Lahu;
+		FIELD l friend Lahu;
+		FIELD m lastAttackedTime I
+	CLASS ana$b StopWanderingGoal
+		FIELD a timer I
+	CLASS ana$c WorriableEntityFilter
+		METHOD a test (Lahl;)Z
+		METHOD a isFriendlyTo (Ljava/util/UUID;)Z
+	CLASS ana$d CalmDownGoal
+		METHOD g isAtFavoredLocation ()Z
+		METHOD h canCalmDown ()Z
+	CLASS ana$e MateGoal
+	CLASS ana$f EatSweetBerriesGoal
+		FIELD g timer I
+		METHOD m eatSweetBerry ()V
+	CLASS ana$g FoxSwimGoal
+	CLASS ana$h FoxData
+		FIELD a type Lana$t;
+		FIELD b uses I
+	CLASS ana$i FoxLookControl
+	CLASS ana$j AttackGoal
+	CLASS ana$k FoxMoveControl
+	CLASS ana$l GoToVillageGoal
+		METHOD g canGoToVillage ()Z
+	CLASS ana$m EscapeWhenNotAggresiveGoal
+	CLASS ana$n JumpChasingGoal
+	CLASS ana$o PickupItemGoal
+	CLASS ana$p SitDownAndLookAroundGoal
+		FIELD c lookX D
+		FIELD d lookZ D
+		FIELD e timer I
+		FIELD f counter I
+		METHOD j chooseNewAngle ()V
+	CLASS ana$q AvoidDaylightGoal
+		FIELD c timer I
+	CLASS ana$r DelayedCalmDownGoal
+		FIELD c timer I
+	CLASS ana$s MoveToHuntGoal
+	CLASS ana$t Type
+		FIELD d byName Ljava/util/Map;
+		FIELD e id I
+		FIELD f key Ljava/lang/String;
+		FIELD g biomes Ljava/util/List;
+		METHOD a getKey ()Ljava/lang/String;
+		METHOD a fromId (I)Lana$t;
+		METHOD a fromBiome (Lbdy;)Lana$t;
+		METHOD a byName (Ljava/lang/String;)Lana$t;
+		METHOD b getBiomes ()Ljava/util/List;
+		METHOD c getId ()I
+	FIELD bC TYPE Lpw;
+	FIELD bD FLAGS Lpw;
+	FIELD bE OWNER Lpw;
+	FIELD bG FRIEND Lpw;
+	FIELD bH PICKABLE_DROP_FILTER Ljava/util/function/Predicate;
+	FIELD bI JUST_ATTACKED_SOMETHING_FILTER Ljava/util/function/Predicate;
+	FIELD bJ CHICKEN_AND_RABBIT_FILTER Ljava/util/function/Predicate;
+	FIELD bK NOTICIBLE_PLAYER_FILTER Ljava/util/function/Predicate;
+	FIELD bL followChickenAndRabbitGoal Lajs;
+	FIELD bM followBabyTurtleGoal Lajs;
+	FIELD bN followFishGoal Lajs;
+	FIELD bO headRollClock F
+	FIELD bP lastHeadRollClock F
```

Comment:
clock -> time?

Reply:
It is not a timer, but something like a ratio (not radian either); used in rendering and updated in `update` method.

Reply:
![image](https://user-images.githubusercontent.com/7806504/52750212-7b373800-2fa0-11e9-8153-b2d00baf5703.png)

Reply:
This is just a smoothening operation for the movement. headRollProgress.

Reply:
Updated.